scripts/build_tools.sh

```diff
@@ -19,4 +19,5 @@ cd $GOPATH/src/github.com/raintank/metrictank/cmd
 for tool in *; do
   cd $tool
   go build -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/$tool
+  cd ..
```

Comment:
FWIW you can just rebase this branch on top of master and then you don't need this commit anymore.